backend/controllers/appointments/bulkAppointmentsUpload.js

```diff
@@ -39,6 +39,17 @@ const bulkAppointmentsUploadFactory = (csvParserService, offenderLoader, logErro
     return uploadedList.filter(x => matchingPrisonerNumbers.indexOf(x) < 0)
   }
 
+  const getLocations = async (context, prisonerList) => {
+    return Promise.all(
+      prisonerList.map(async prisoner => {
+        const prisonerToUpdate = prisoner
+        const location = await offenderLoader.offenderLocation(context, prisoner.assignedLivingUnitId)
+        prisonerToUpdate.cellNo = location.description
+        return prisonerToUpdate
```

Comment:
I think this could be simplified with destructuring:
```
prisonerList.map(async prisoner => {
        const location = await offenderLoader.offenderLocation(context, prisoner.assignedLivingUnitId)
        return {
                     ...prisoner,
                     cellNo: location.description
         }
```

I've not run the above, mind you, but something like it.